autoload/kite/async.vim

```diff
@@ -93,12 +97,13 @@ function! s:on_stdout_vim(_channel, data) dict
   call add(self.stdoutbuffer, a:data)
 endfunction
 
-
-function! s:on_close_vim(channel) dict
+function! s:on_exit_vim(job, exit_status) dict
+  " Allow time for any buffered data to trigger out_cb.
+  " 5m is an educated guess.
+  sleep 5m
```

Comment:
is `5m` 5 milliseconds or something else?

Reply:
5 milliseconds.

It's possible for Vim to call the job's exit callback before all the data from stdout has been fed to the stdout callback.  I think this is something Vim should hide from the user and deal with, but instead the docs advise adding a "short" sleep at the top of the exit callback.

If the sleep is too short then it's possible to miss some data at the end from stdout.  I think 1ms would be enough time so 5ms really should be safe.